observations/forward_operators/obs_def_abs_humidity_mod.f90

```diff
@@ -0,0 +1,186 @@
+! DART software - Copyright UCAR. This open source software is provided
+! by UCAR, "as is", without charge, subject to all terms of use at
+! http://www.image.ucar.edu/DAReS/DART/DART_download
+!
+! $Id$
```

Comment:
We can remove these two lines ... the Id was svn boilerplate that is not needed with git.